incubator/kubeless/templates/kafka-controller-deployment.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.kafkaTrigger.enabled }}
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    kubeless: kafka-trigger-controller
+  name: kafka-trigger-controller
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    matchLabels:
+      kubeless: kafka-trigger-controller
+  template:
+    metadata:
+      labels:
+        kubeless: kafka-trigger-controller
+    spec:
+      containers:
+      - image: "{{ .Values.kafkaTrigger.deployment.image.repository }}:{{ .Values.kafkaTrigger.deployment.image.tag }}"
+        imagePullPolicy: {{ .Values.kafkaTrigger.deployment.image.pullPolicy }}
+        name: kafka-trigger-controller
+        env:
+        - name: KAFKA_BROKERS
+          value: {{ .Values.kafkaTrigger.env.kafkaBrokers }}
```

Comment:
can you add this value as required? (like https://github.com/kubernetes/charts/blob/d284317c2258f4e9d6a870d79db1e4095e441ac3/stable/lamp/templates/secret.yaml#L33)

Reply:
Sure thing. Is it required conditionally?
i.e. if I `kafkaTrigger.enabled = false` will `kafkaTrigger.env.kafkaBrokers` be required?

Reply:
in that case it is not required, I am not sure if that is something doable